pyproject.toml

```diff
@@ -45,11 +45,11 @@ Jinja2 = "^3.0.3"
 black = "*"  # To format files in cli tools
 prometheus-client = "^0.16.0"
 aiohttp = { version = "^3.8.4", optional = true }
-aiolimiter = { version = "^1.1.0", optional = true }
-aiobotocore = { version = "^2.5.2", optional = true }
-aiofiles = { version = "^23.1.0", optional = true }
-aiocsv = { version = "^1.2.4", optional = true }
-loguru = { version = "^0.7.0", optional = true }
+aiolimiter = "^1.1.0"
+aiobotocore = "^2.6.0"
+aiofiles = "^23.2.1"
+aiocsv = "^1.2.4"
+loguru = "^0.7.2"
```

Comment:
You should keep it as optional

Reply:
While optional it wasn't installed when I runned `poetry install` and the tests didn't pass since they do require those libraries, I could skip the tests but it made me think this was the best option; am I missing something?

Reply:
Yes that's the point, to not bring those packages in the modules that don't need it.

You can install them in local using `poetry install --extras=all`.